app/views/records/edit_fields/_workType.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= f.input key,
-            as: :controlled_vocabulary,
+            as: :single_value_controlled_vocabulary,
```

Comment:
`:workType` is obligation {0, n} on the MAP.
Try `:license`, `:access_restrictions`, or `:repository`

Reply:
Or are you thinking of `:resource_type`?